tests/test_migrators.py

```diff
@@ -768,13 +768,15 @@
 build:
   number: 1
 
+  merge_build_host: True  # [win]
   rpaths:
     - lib/R/lib/
     - lib/
   skip: True  # [win32]
 
 requirements:
   build:
+    - {{posix}}zip               # [win]
```

Comment:
This line shouldn't be removed. It needs to be there for the recipe to work on windows. (i.e. if the recipe is built on windows and not on unix like we do in CI)

Reply:
Isn't this line being added?

Reply:
selectors aren't allowed for noarch

Reply:
> @mariusvniekerk, technically they are. It's only selectors in the run dependencies that should not be used.

Perhaps we need to update the linter....

Reply:
So the interest here is making sure end users can build it easily. Is that right? How is it being used? Would it be possible to integrate this more into the tooling somehow to avoid duplication across recipes?